app/views/publisher_mailer/verified_no_wallet.html.slim

```diff
@@ -0,0 +1,19 @@
+p.salutation= t("publisher_mailer.shared.salutation", name: @publisher.name)
+
+== t("publisher_mailer.verified_no_wallet.body_html_1", formatted_url: @publisher_dashboard_url.gsub(/http(s?):\/\//, '').gsub(/\/$/, ''), url: @publisher_dashboard_url)
+
+div.center= link_to(t("publisher_mailer.verified_no_wallet.button"), @publisher_dashboard_url, class: "btn-primary btn-login")
+
+== t("publisher_mailer.verified_no_wallet.body_html_2")
+
+p= t("publisher_mailer.shared.signature")
+
+hr
+
+.hint
+  p
+    span= t("publisher_mailer.shared.button_not_working")
+    span= link_to(@publisher_dashboard_url, @publisher_dashboard_url, class: "fallback-link", onMouseOver: "this.style.color='#48494b'", onMouseOut: "this.style.color='#818589'")
```

Comment:
I have a suspicion that the `onMouseOver`, being JavaScript, does not actually run in any email clients.

Reply:
This was copied from here:
https://github.com/brave-intl/publishers/blob/f7385755f1b4b81e850c5bdaeedd8944016f3418/app/views/publisher_mailer/login_email.html.slim#L19

I tested in both macOS Mail and Gmail, and neither seem to execute the JS hover effect. I believe that `a:hover` can possibly be achieved in email via a CSS `<style>` block. Maybe we could try that?

cc: @lgebhardt

Reply:
I agree we should probably remove the JS. It was intended for webmail clients, but it just adds complexity for very little gain. CSS might be a good way to go for the hover. It should work in gmail now. But there are still enough clients that strip style tags that we don't want to use it for anything critical.